src/elements/Header/_Header.js

```diff
@@ -1,42 +1,98 @@
 import cx from 'classnames'
-import React, { createElement, Component, PropTypes } from 'react'
-
+import React, { PropTypes } from 'react'
 import META from '../../utils/Meta'
+import {
+  getUnhandledProps,
+  iconPropRenderer,
+  imagePropRenderer,
+  useKeyOnly,
+} from '../../utils/propUtils'
+
+function _Header(props) {
+  const {
+    _sdClass, _headerElement,
+    left, center, right, justified, dividing, block, attached, floating,
+    icon, image, children, className,
+  } = props
+
+  const classes = cx(
```

Comment:
That is a great idea and it's been on my list.  Thanks for trudging through this in it's absence.

I have started some notes in the actual code along side the prop utils themselves here:
https://github.com/TechnologyAdvice/stardust/blob/master/src/utils/propUtils.js#L76

In my head I see a simple markdown file that explains to folks "How to write a component".  One of the first orders of business in that doc is the explanation of the component API and the 4 different prop to className permutations (noted in the code linked above).

I think even starting a markdown file with just that ^ information in it for now is a good start.  We can iterate and add as we go.  I'll keep this on my list, but if you get to it before me I'll gladly review and contribute.